examples/train.py

```diff
@@ -8,20 +8,20 @@
 
 from dfibert.tracker.nn.rl import Agent, Action_Scheduler
 
-import dfibert.envs.RLtractEnvironment as RLTe
+import dfibert.envs.tractography as RLTe
 
 
 def train(path, max_steps=3000000, replay_memory_size=20000, eps_annealing_steps=100000, agent_history_length=1, evaluate_every=20000, eval_runs=5, network_update_every=10000, max_episode_length=200, learning_rate=0.0000625):
```

Comment:
documentation missing but would be really helpful. @phanfeld might probably provide more insights in the trainign algorithm etc